### DISCUSSION TITLE
My first thoughts on Zitadel after a few struggling

### DISCUSSION THREAD

**Discussion by Badisi** [zitadel/zitadel#4299] [Ideas]
I know the product is still in heavy development so I thought I would share with you guys my personal experience with Zitadel. 

Especially because it was my first attempt to use it and I was expecting something more "user friendly" than what it actually is.

These are constructive comments and I hope you will find them useful 😊

---

1. `Organisation name` vs `Domain`
    - I went to https://zitadel.cloud to create a new account
    - The **Organization Name** field was mandatory (but I wanted to sign-up as an individual, so I used the same value as **Username**)
    - Once created and logged-in I realized that the Organization name was used as the domain name, which is not what I would have wanted!
    - **Issue 1**: there is no way I can rename the default domain
    - **Issue 2**: I now have a domain named the same as my username, so whenever I see it writing somewhere I don't know whether it is about me or my domain

2. `console` vs `admin`
    - Once you created your account, you are logged in: `zitadel.cloud/ui/console` but here there is no clear info on what to do next or where to go (ie. I was looking for a way to create a client, like you are supposed to do in an IDP, but couldn't find it)
    - I finally found (but don't even remember how) the `zitadel.cloud/admin` interface where I could create an **Instance**
    - I was asked for a **name** but again I didn't know this would involve the creation of another domain with that name
    - **Issue 3**: an instance cannot be renamed (as well as its associated domain)
    - **Issue 4**: there is no clearly links between console and admin

3. `instance - password renewal`
    - Once I had my instance created, I clicked on it and I was asked to log-in, so I used the account I previously created
    - There I was asked for a code (received by email) so I could set my new password.... ?!!
    - I decided to re-enter the same password as during my account creation and finally land to my instance console
    - **Issue 5**: why would I change my password ? did it created another user ??

4. `console vs console vs console`
    The **instance console** looks the same as my **account console** which can be nice but get the user completely lost instead. Every time I'm on a page, it is hard to identify quickly and simply where I am (ie. which consoles, portals, etc. ?).
    - They all have the same look and feel
    - They all have notions of `Organizations`, `Domains` (on the Instance you even have **Organizations** as tab + a section **Organization** under the instance tab too). That makes me feel they are everywhere and drives me crazy.
    - The instance console also have a toggle button on the top right of the screen (ie. Organization/Instance) where you have no idea if it is actually enabled or the opposite and brings kind of another console!
    - All the consoles/portals have the same starting name and tab icon in my web browser (ie. "ZITADEL - ..."), which makes them hard to identify
    - **Issue 6**: a total of 3 consoles !! (account, instance and org), hard to distinguish + 1 customer portal -> that is clearly too much to deal with, especially if they look exactly the same

5. `a clear lack of removal`
    - **Issue 7**: not possible to delete the default domain (in case you need to recreate/reset it)
    - **Issue 8**: not possible to delete an instance
    - **Issue 9**: not possible to delete my user account

7. `missing contact`
    - **Issue 10**: After all these strugglings, I was looking for a **contact/support** section and found none on the consoles (but one on the customer portal), so I had to go the social way , find the GitHub and then the `Discussions`.

**Comment by juergrinaldi** [comment]
Ahoi @Badisi ,

First of all: Thanks a million for the super constructive feedback. It's much apprechiated and we will definitely have a closer look on these.

As you already mentioned we are constantly developing and improving ZITADEL, so some of your issues might get tackled in the near future. Especially the the onboarding process in high focus right now.

To tackle some of your issues right away:

Issue 1/2: Yes, that's currently not possible, as it would have high impact system-wise. But you can easily create a new instance with a more suiting name. (but we will have a look on how to make that clear in the first place)

Issue 3: This will be fixed in the upcoming update.

Issue 4: This should get more clear with the upcoming update.

Issue 5/6: Console (general settings) and Admin (instances/accounting) are indeed two separated applications and with that you're having different users for them. That's an issue that is currently heavily discussed internally. We also have done some rework that should ease this issue a little in the meantime.
The same look and feel on these is indeed intentionally, as it shall solidify trust and familiarity in the UX. But we can see that there might be an issue in identification for these applications. I'll definitely see if we can have that fixed.

Issue 7/8/9: We have that tackled in our internal build and it should be fixed very soon.

Issue 10: Oh, good catch! I've just created an [issue ](https://github.com/zitadel/zitadel/issues/4315) to fix that. 

Additionally here are two issues that are worked on right now regarding the onboarding process (login flow):
https://github.com/zitadel/zitadel/issues/2942
https://github.com/zitadel/zitadel/milestone/23

Thanks again for the valuable inputs Badisi and let us know if we can anything!

**Comment by davidklemme** [comment]
Just want to second issue 5/6

for a minute i thought something was broken and it caused quite some confusion ;)
